_blog/2019/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: 2019 Posts
+overview: Blog posts for 2019
+
+order: 20
```

Comment:
`order: 9`
newer posts should appear at the top.

Reply:
Fixed in the new commit